aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -720,6 +728,55 @@ module aptos_token::token {
         token_data.uri = uri;
     }
 
+    public fun mutate_tokendata_royalty(creator: &signer, token_data_id: TokenDataId, royalty_points_numerator: u64, royalty_points_denominator: u64, payee_address: address) acquires Collections {
```

Comment:
since you already have `create_royalty` function, how about just make this function `public fun mutate_tokendata_royalty(creator: &signer, token_data_id: TokenDataId, royalty: Royalty)` as you previously did.

Reply:
love this.